composer.json

```
@@ -4,24 +4,28 @@
     "type": "project",
     "license": "proprietary",
     "require": {
-        "php": "^8.0",
+        "php": "^8.2",
         "dvsa/mot-cpms-notifications": "^1.0.0",
```

Can we match the 8.2 php version?
```suggestion
        "dvsa/mot-cpms-notifications": "dev-feature-bl-17032-php-82-update",
```